### ISSUE TITLE
+-prefixed odometer breaks it

### ISSUE THREAD

**Issue by GoogleCodeExporter** [evancharlton/android-mileage#13]
```
What steps will reproduce the problem?
1. Add a fillup with odometer '+500'
2. Save

Expected behavior:
Saved with odometer +500

Actual behavior:
E/AndroidRuntime(17719): FATAL EXCEPTION: main
E/AndroidRuntime(17719): java.lang.NullPointerException
E/AndroidRuntime(17719):    at 
com.evancharlton.mileage.FillupActivity.setFields(FillupActivity.java:317)
E/AndroidRuntime(17719):    at 
com.evancharlton.mileage.BaseFormActivity$1.onClick(BaseFormActivity.java:50)
E/AndroidRuntime(17719):    at android.view.View.performClick(View.java:2485)
E/AndroidRuntime(17719):    at android.view.View$PerformClick.run(View.java:9080)
E/AndroidRuntime(17719):    at android.os.Handler.handleCallback(Handler.java:587)
E/AndroidRuntime(17719):    at android.os.Handler.dispatchMessage(Handler.java:92)
E/AndroidRuntime(17719):    at android.os.Looper.loop(Looper.java:123)
E/AndroidRuntime(17719):    at 
android.app.ActivityThread.main(ActivityThread.java:3647)
E/AndroidRuntime(17719):    at java.lang.reflect.Method.invokeNative(Native 
Method)
E/AndroidRuntime(17719):    at java.lang.reflect.Method.invoke(Method.java:507)
E/AndroidRuntime(17719):    at 
com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:839)
E/AndroidRuntime(17719):    at 
com.android.internal.os.ZygoteInit.main(ZygoteInit.java:597)
E/AndroidRuntime(17719):    at dalvik.system.NativeStart.main(Native Method)


Mileage version: 3.0.2
Android version:
Device:

Please use labels and text to provide additional information.

```

Original issue reported on code.google.com by `evancharlton` on 4 Mar 2011 at 4:22

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 3.0.3
```

Original comment by `evancharlton` on 4 Mar 2011 at 4:40
- Changed state: **Verified**

**Comment by GoogleCodeExporter** [comment]
Original comment by `evancharlton` on 17 Apr 2011 at 8:21
- Changed state: **Released**
